README.md

```diff
@@ -66,6 +66,7 @@ Creates a new `RetryOperation` object. `options` is the same as `retry.timeouts(
 
 * `forever`: Whether to retry forever, defaults to `false`.
 * `unref`: Whether to [unref](https://nodejs.org/api/timers.html#timers_unref) the setTimeout's, defaults to `false`.
+* `maxRetryTime`: The number of milliseconds maximum amount to retry the operation. Default is `Infinity`  
```

Comment:
Let's reword this to "The maximum time (in milliseconds) that the retried operation is allowed to run. Default is `Infinity`".